marc_to_solr/lib/traject_config.rb

```diff
@@ -727,6 +728,10 @@
 #    546 XX 3a
 to_field 'language_display', extract_marc('5463a')
 
+to_field 'language_name_display' do |record, accumulator|
```

Comment:
@sandbergja can you add a comment that this field will be used to show specific languages in the record page?

Reply:
I created one. https://github.com/pulibrary/bibdata/issues/2100